data/low_income_thresholds.json

```diff
@@ -405,6 +402,24 @@
         "8": 115800
       }
     },
+    "nv-nv-energy-residential-air-conditioning": {
+      "incentives": [
+        "NV-23",
+        "NV-24",
+        "NV-25"
+      ],
+      "source_url": "https://dwss.nv.gov/Energy/3_Eligibility_Criteria/",
```

Comment:
This program doesn't specifically list these thresholds, but on their [website](https://www.nvenergy.com/save-with-powershift/home-energy-saver/residential-ac-and-mid-stream/residential-air-conditioning) they say: "Customers who currently participate in the Nevada Energy Assistance Program are automatically eligible to receive a higher incentive. Customers living in affordable housing properties listed under organizations such as Southern Nevada Regional Housing Authority (SNRHA), Reno Housing Authority (RHA), Nevada Rural Housing Authority (NRHA), Housing and Urban Development (HUD), and Section 8 Housing may also qualify for this increased incentive."

To simplify this, I found the thresholds for the Nevada Energy Assistance Program, which are listed here

Reply:
I think there's an argument to be made for extending the `low_income` field somehow to express eligibility "by reference" to other programs -- i.e. if you're already eligible for LIHEAP or Section 8 or whatever, then you're eligible for this. But what you've done here is fine.